### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by Zero3K** [Genymobile/feedback#29] [Genymotion Desktop] [answered]
It would be nice if you all made a Discord Server for talking about it.

**Comment by agavignet** [comment, marked as answer]
Thanks for your suggestion, we're going to think about it :)

**Reply by Zero3K** [reply]
Any news?

**Comment by Zero3K** [comment]
I can make one and add you (and anyone else in the team) as an admin/moderator. Is that okay?

**Reply by ohlalayeah** [reply]
they are going to think about it means they won't do it.
